runtime/darwinia/src/lib.rs

```diff
@@ -535,19 +543,33 @@ impl_runtime_apis! {
 			Staking::power_of_rpc(account)
 		}
 	}
+
+	#[cfg(feature = "try-runtime")]
+	impl frame_try_runtime::TryRuntime<Block> for Runtime {
+		fn on_runtime_upgrade() -> Result<(Weight, Weight), sp_runtime::RuntimeString> {
+			log::info!("try-runtime::on_runtime_upgrade Darwinia.");
+			let weight = Executive::try_runtime_upgrade()?;
+			Ok((weight, RuntimeBlockWeights::get().max_block))
+		}
+	}
 }
 
-pub struct CustomOnRuntimeUpgrade;
-impl darwinia_elections_phragmen::migrations_2_0_0::ToV2 for CustomOnRuntimeUpgrade {
-	type AccountId = AccountId;
-	type Balance = Balance;
-	type Module = ElectionsPhragmen;
+impl pallet_babe::migrations::BabePalletPrefix for Runtime {
+	fn pallet_prefix() -> &'static str {
+		"Babe"
+	}
 }
-impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
-	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		// --- substrate ---
-		// use frame_support::migration::*;
 
-		darwinia_elections_phragmen::migrations_2_0_0::apply::<Self>(5 * MILLI, 100 * MILLI)
+pub struct CustomOnRuntimeUpgrade;
+impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<(), &'static str> {
+		darwinia_staking::migrations::v6::pre_migrate::<Runtime>()
+	}
+
+	fn on_runtime_upgrade() -> Weight {
+		pallet_babe::migrations::add_epoch_configuration::<Runtime>(BABE_GENESIS_EPOCH_CONFIG);
```

Comment:
```suggestion
		pallet_babe::migrations::add_epoch_configuration::<Runtime>(
		             allowed_slots: AllowedSlots::PrimaryAndSecondaryPlainSlots,
			     ..BABE_GENESIS_EPOCH_CONFIG);
```

According to the information from @andresilva and @AurevoirXavier , this should be changes to `PrimaryAndSecondaryPlainSlots`

References:
https://github.com/darwinia-network/darwinia/pull/641/files#diff-c23cc93db5b2f8c1b19170f6ce738217642392f312028b87011d240406b255b3L436

https://github.com/paritytech/polkadot/pull/2467